source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -696,11 +694,9 @@ export class CardDesigner extends Designer.DesignContext {
                                     dialog.close()
                                     this.launchImagePopup()
                                 },
-                                onKeyEvent: (e: KeyboardEvent) => {
-                                    if (e.key === ControlConstants.keys.enter) {
-                                        dialog.close();
-                                        this.launchImagePopup();
-                                    }
+                                onKeyEnterEvent: (e: KeyboardEvent) => {
```

Comment:
why do you need this change? Will customer be impacted and a breaking change?

Reply:
I added this to avoid adding the arrow key handlers in this method as well. And so that if users decide to override the key event, the arrow handling will still work.

This isn't a breaking change - I just added `onKeyEvent` in this PR (https://github.com/microsoft/AdaptiveCards/pull/7710) so it hasn't shipped